From 850ef6c884f24773abced96ba65cf497aae946e1 Mon Sep 17 00:00:00 2001
From: Allen Winter <winter@kde.org>
Date: Thu, 2 Oct 2025 09:10:21 -0400
Subject: [PATCH] CMakeLists.txt - remove trailing '/' to fix CMP0177 warning

KDE_MODULES_INSTALL_DIR needs to be normalized for
the install(PROGRAMS) command. The easiest solution
is to not have a trailing '/'.

Fixes:
```
CMake Warning (dev) at CMakeLists.txt:96 (install):
Policy CMP0177 is not set: install() DESTINATION paths are normalized.
```
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index a8de8636..2896268b 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -40,7 +40,7 @@ set(SHARE_INSTALL_DIR share
 set(CMAKECONFIG_INSTALL_DIR       ${SHARE_INSTALL_DIR}/ECM/cmake/)
 
 set(FIND_MODULES_INSTALL_DIR      ${SHARE_INSTALL_DIR}/ECM/find-modules/)
-set(KDE_MODULES_INSTALL_DIR       ${SHARE_INSTALL_DIR}/ECM/kde-modules/)
+set(KDE_MODULES_INSTALL_DIR       ${SHARE_INSTALL_DIR}/ECM/kde-modules)
 set(MODULES_INSTALL_DIR           ${SHARE_INSTALL_DIR}/ECM/modules/)
 set(TOOLCHAIN_MODULES_INSTALL_DIR ${SHARE_INSTALL_DIR}/ECM/toolchain/)
 
-- 
GitLab

